packages/manager/modules/pci/src/projects/project/private-registry/create/create.routing.js

```diff
@@ -0,0 +1,34 @@
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider
+    .state('pci.projects.project.private-registry.create', {
+      url: '/create',
+      views: {
+        modal: {
+          component: 'pciProjectPrivateRegistryCreate',
+        },
+      },
+      params: {
+        fromState: null,
+      },
+      layout: 'modal',
+      resolve: {
+        goBack: /* @ngInject */  goBackToState => goBackToState,
+      },
+    })
+    .state('pci.projects.project.private-registry.onboarding.create', {
```

Comment:
I think you should have one state declaration per routing file, this means taht `pci.projects.project.private-registry.onboarding.create` should be declared in `/onboarding/create` and just reusing the `pciProjectPrivateRegistryCreate` component